portfolio/src/main/java/Comments.java

```diff
@@ -0,0 +1,34 @@
+package usercomments;
+
+public class Comments{
```

Comment:
nit: Singular form (Comment) since this class represents a single comment.

Reply:
Done!